Examples/Example/Source/Core/View Controllers/ArrowOrientationLeftAndRightViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
```

Comment:
Here, you need to use the same MIT header as the other files, rather than the default one generated by Xcode.

Reply:
done